cli/cliui/table.go

```diff
@@ -67,7 +67,7 @@ func DisplayTable(out any, sort string, filterColumns []string) (string, error)
 	}
 
 	// Get the list of table column headers.
-	headersRaw, err := typeToTableHeaders(v.Type().Elem())
```

Comment:
looks like you might want to rebase from main to remove these commits

Reply:
yes i need to remove these my bad.